fig-flag.svg

```diff
@@ -0,0 +1,136 @@
+       x="111.28967"
+       y="287.57559"
+       id="text829-67"><tspan
+         sodipodi:role="line"
+         id="tspan827-5"
+         x="111.28967"
+         y="287.57559"
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:19.75555611px;font-family:Sans-serif;-inkscape-font-specification:'Sans-serif Bold';stroke-width:0.26458332px">&lt;1cm</tspan></text>
+    <path
+       style="fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+       d="m 152.64021,-145.14494 v 14.59185 h 192.40492 v -16.34287"
+       id="path825-3-35"
+       inkscape:connector-curvature="0" />
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:10.58333302px;line-height:6.61458349px;font-family:Sans-serif;-inkscape-font-specification:'Sans-serif Bold';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       x="219.86436"
+       y="-106.569"
+       id="text829-6-6"><tspan
+         sodipodi:role="line"
+         id="tspan827-7-2"
+         x="219.86436"
+         y="-106.569"
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:19.75555611px;font-family:Sans-serif;-inkscape-font-specification:'Sans-serif Bold';stroke-width:0.26458332px">20cm</tspan></text>
+    <path
+       style="fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+       d="m 154.99644,251.07011 h 13.89259 V -146.6781 h -15.5597"
+       id="path825-3-9"
+       inkscape:connector-curvature="0" />
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:10.58333302px;line-height:6.61458349px;font-family:Sans-serif;-inkscape-font-specification:'Sans-serif Bold';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       x="181.94379"
+       y="50.795963"
+       id="text829-6-1"><tspan
+         sodipodi:role="line"
+         id="tspan827-7-27"
+         x="181.94379"
+         y="50.795963"
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:19.75555611px;font-family:Sans-serif;-inkscape-font-specification:'Sans-serif Bold';stroke-width:0.26458332px">40cm</tspan></text>
```

Comment:
We should be consistent about our units -- the text describing the spec uses mm, so the diagram should too.

Reply:
fixed in 96616922a530f79b2f0f256625f4c4a48c1a6227

Reply:
fixed in 96616922a530f79b2f0f256625f4c4a48c1a6227